keyboards/skyloong/gk980/bl/matrix.c

```diff
@@ -0,0 +1,166 @@
+// Copyright 2023 NaturaZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+//Matrix read rows set col:DIODE_DIRECTION == ROW2COL
+
+#include "quantum.h"
+#include "debounce.h"
```

Comment:
```suggestion
```

Reply:
Do not include debounce.h, how to implement the key debounce?

Reply:
You don't need to if using custom matrix lite. Please read the docs.